index2.html

```diff
@@ -0,0 +1,1253 @@
+        var timesRead = (msg.data.length > 0) ? msg.data[0].json().timesRead : 0;
+        if (msg.writeable)
+          adapter.write([{ timesRead: (timesRead + 1) % 1000 }], scope);
+        else
+          console.log("Read-only tag; value = " + timesRead);
+      };
+
+      adapter.watch(scope).then(function() {
+        console.log("We are now watching tags via NFC");
+      });
+    }).catch(function(err) {
+      console.log("No NFC adapters found");
+    });
+  </pre>
+  <pre title="Save and restore game progress with another device"
+       class="example highlight">
+    navigator.nfc.findAdapters().then(function(adapters) {
+      var adapter = adapters[0];
+      var scope = { scope: "/example.com/mygame" };
+
+      adapter.onread = (event) => {
+        console.log("Game state received from: " + event.scope);
+        console.log("Game state: " + event.data.json());
+      };
+
+      adapter.onbeforepush = (event) => {
+        var msg =  [ JSON.stringify({ level: 3, points: 4500, lives: 3 }) ];
+        adapter.setPushMessage(msg, scope).then(function() {
+          watcher.onmessagepushed = () => {
+          alert("We successfully pushed a message");
+        });
+      };
+
+      adapter.watch(scope).then(function() {
+        console.log("We are now watching game progress via NFC");
+      });
+    });
+  </pre>
+</section> <!-- Usage examples -->
+
+
+<!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
+<section> <h2>Security and Privacy Considerations</h2>
+  <p>
+    <a title="user agent">User agents</a> MUST NOT provide Web NFC API access
+    to <a>web app</a>s without the
+    <a>expressed permission</a> of the user. User agents must acquire consent
+    for permission through a user interface for each call to the request*()
+    methods, unless a prearranged trust relationship applies.
+  </p>
+  <p>
+    User agents may support prearranged trust relationships that do not require
+    such per-request user interfaces.
+  </p>
+  <p class="note">
+    Considering adding the following:
+    User agents MUST implement the WebNFC API to be HTTPS-only. SSL-only support
+    provides better protection for the user against man-in-the-middle attacks
+    intended to obtain push registration data. Browsers may ignore this rule
+    for development purposes only.
+  </p>
+  <p>
+    Permissions that are preserved beyond the current browsing session must be
+    revocable.
+  </p>
+</section> <!-- Security and Privacy  -->
+
+<!-- - - - - - - - - - - Extended interface Navigator - - - - - - - - - - - -->
+<section> <h2>Extensions to the <strong>Navigator</strong> interface</h2>
+  <p>
+    The HTML specification defines a
+    <a href="http://www.whatwg.org/specs/web-apps/current-work/#dom-navigator">
+    <code>Navigator</code></a> interface [HTML] which this specification
+    extends.
+  </p>
+  <dl title="partial interface Navigator" class="idl">
+    <dt>readonly attribute NFC nfc</dt>
```

Comment:
For the single one meta-object NFC it could be the full acronym, but for compund names it's NfcSomething.